server/.eslintrc.json

```diff
@@ -34,6 +34,7 @@
       "error",
       "always"
     ],
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
shouldn't we avoid using the any type?

Reply:
I modified the validation to return all encountered errors during validation process.

I made quite a few changes so I think someone else should review these.